test/bugs-reported/bug6.chpl

```diff
@@ -0,0 +1,14 @@
+use Queue;
```

Comment:
I did not know that multiple files can define same module and the programmer can choose which to use. Thanks @lydia-duncan for clarifying this. However, I am still unable to compiler this. Can you please direct me to a resource that explains how to do this?

@benharsh Removing this bug.